QRCoderTests/PostscriptQRCodeRendererTests.cs

```diff
@@ -9,7 +9,7 @@ public void can_render_postscript_qrcode_simple()
         var gen = new QRCodeGenerator();
         var data = gen.CreateQrCode("This is a quick test! 123#?", QRCodeGenerator.ECCLevel.L);
         var ps = new PostscriptQRCode(data).GetGraphic(5);
-        RemoveCreationDate(ps).ShouldMatchApproved(x => x.NoDiff());
+        ps.ShouldMatchApproved(x => x.NoDiff());
```

Comment:
Tests previously removed the creation date from the verified files.  So all tests continue to match with no change to test result data.